src/test/java/org/mockito/internal/stubbing/defaultanswers/ReturnsEmptyValuesTest.java

```diff
@@ -108,7 +108,64 @@ public void should_return_empty_optional() throws Exception {
     }
 
     @Test
-    public void should_return_empty_stream() throws Exception {
+    public void should_return_empty_OptionalDouble() throws Exception {
+        Class<?> streamType = getClassOrSkipTest("java.util.stream.DoubleStream");
+
+        //given
+        Object stream = mock(streamType);
+        Object optional = streamType.getMethod("findAny").invoke(stream);
+        assertNotNull(optional);
+        assertFalse((Boolean) Class.forName("java.util.OptionalDouble").getMethod("isPresent").invoke(optional));
+
+        Invocation findAny = this.getLastInvocation();
+
+        //when
+        Object result = values.answer(findAny);
+
+        //then
+        assertEquals(optional, result);
+    }
+
+    @Test
+    public void should_return_empty_OptionalInt() throws Exception {
```

Comment:
JUnitParams is fine, I have nothing against, let's go for it, test readability and maintainability is just as important as production code.